tests/test_components/test_autograd.py

```diff
@@ -58,16 +58,17 @@
 WVL = 1.0
 FREQ0 = td.C_0 / WVL
 FREQS = [FREQ0]
+FWIDTH = FREQ0 / 10
 
 # sim sizes
 LZ = 7 * WVL
 
 # NOTE: regular stuff is broken in 2D need to change volume and face integration to handle this
-IS_3D = True
+IS_3D = False
```

Comment:
Is the comment still relevant if 2d is default now?

Reply:
no, stuff is fixed in 2D, and also I forgot to make 3D default so will change it back, thanks